docs/reference/content/getting-started/quick-tour-admin.md

```diff
@@ -0,0 +1,178 @@
+that can be found with the [driver source]({{< srcref "examples/src/test/scala/tour/QuickTourAdmin.scala">}}). 
+
+{{% note %}}
+See the [installation guide]({{< relref "getting-started/installation-guide.md" >}}) for instructions on how to install the MongoDB Driver.
+
+This guide uses the `Helper` implicits as covered in the [Quick Tour Primer]({{< relref "getting-started/quick-tour-primer.md" >}}).
+{{% /note %}}
+
+## Setup
+
+To get started we'll quickly connect and create a `mongoClient`, `database` and `collection`
+variable for use in the examples below:
+
+```scala
+val mongoClient: MongoClient = MongoClient()
+val database: MongoDatabase = mongoClient.getDatabase("mydb")
+val collection: MongoCollection[Document] = database.getCollection("test")
+```
+
+{{% note %}}
+Calling the `getDatabase()` on `MongoClient` does not create a database.
+Only when a database is written to will a database be created.  Examples include the creation of an index or the insertion of a document 
+into a previously non-existent collection.
+{{% /note %}}
+
+## Get A List of Databases
+
+You can get a list of the available databases by calling the `listDatabaseNames` method.  Here we use the implicit 
+`printResults` helper so that we can print the list of database names:
+
+```scala
+mongoClient.listDatabaseNames().printResults()
+```
+
+## Drop A Database
+
+You can drop a database by name using a `MongoClient` instance. Here we block for the `Observable` to complete before continuing.
+
+```scala
+mongoClient.getDatabase("databaseToBeDropped").drop().headResult()
+```
+
+## Create A Collection
+
+Collections in MongoDB are created automatically simply by inserted a document into it. Using the 
+[`createCollection`]({{< apiref "org.mongodb.scala.MongoDatabase@createCollection(collectionName:String):org.mongodb.scala.Observable[org.mongodb.scala.Completed]">}}) method, 
+you can also create a collection explicitly in order to to customize its configuration. For example, to create a capped collection sized to 1 megabyte:
+
+```scala
+database.createCollection("cappedCollection",
+  CreateCollectionOptions().capped(true).sizeInBytes(0x100000)
+).printHeadResult("Collection Created! ")
+```
+
+## Get A List of Collections
+
+You can get a list of the available collections in a database:
+
+```scala
+database.listCollectionNames().printResults("Collection Names: ")
+```
+
+## Drop A Collection
+
+You can drop a collection by using the drop() method:
+
+```scala
+collection.drop().headResult()
+```
+
+## Create An Index
+
+MongoDB supports secondary indexes. To create an index, you just
+specify the field or combination of fields, and for each field specify the direction of the index for that field.
+For `1` ascending  or `-1` for descending. The following creates an ascending index on the `i` field by using the 
+[`Sorts.ascending`]({{< apiref "org.mongodb.scala.model.Sorts$@ascending(fieldNames:String*):org.bson.conversions.Bson">}}) helper:
```

Comment:
This link resolves to http://rosslawley.co.uk/mongo-scala-driver/1.0/api/index.html#org.mongodb.scala.model.Sorts$@ascending(fieldNames:String*):org.bson.conversions.Bson and 404s.